README.md

```diff
@@ -27,8 +27,10 @@
  
  
 <!-- DEMO --> 
-## Demo
-<a>
+## Explanation?
+RPC (Rich Presence) is what allows discord to show others what you are currently doing within supported applications!

```

Comment:
I was thinking more like:
`
## About
Valorant RPC (Rich Presence) allows you to show in-game details such as the current score in your Discord Profile!
`